mkdocs/docs/metrics/defining-iter8-metrics.md

```diff
@@ -0,0 +1,640 @@
+---
+template: main.html
+---
+
+# Defining Iter8 Metrics
+
+This document describes how you can create Iter8 metrics and (optionally) supply authentication information that may be required by the metrics provider.
+
+Metric providers differ in the following aspects.
+
+* HTTP request authentication method: no authentication, basic auth, API keys, or bearer token
+* HTTP request method: GET or POST
+* Format of the JSON response returned by the provider
+* The `jq` expression used by Iter8 to extract the metric value from the JSON response
```

Comment:
Also the query string or body of the request.

Reply:
Done.